src/components/data-exploration/differential-expression-tool/DiffExprCompute.jsx

```diff
@@ -285,14 +285,60 @@ const DiffExprCompute = (props) => {
         <Radio
           style={radioStyle}
           value={ComparisonType.WITHIN}>
-          Compare cell sets within a sample/group
+          <Space>
+            Compare cell sets within a sample/group
+            <Tooltip overlay={(
+              <span>
+                This is useful for finding differentially expressed genes when comparing cell sets (e.g. for finding marker genes that distinguish one cluster from another). The calculation uses the presto implementation of the Wilcoxon rank sum test and auROC analysis. For more information see the
```

Comment:
```suggestion
                This is useful for finding marker genes that distinguish one cluster from another. The calculation uses the presto implementation of the Wilcoxon rank sum test and auROC analysis. For more information see the
```
This is not typically called differential expression but rather marker gene detection